src/core/ext/transport/cronet/transport/cronet_transport.cc

```diff
@@ -314,14 +333,8 @@ static void add_to_storage(struct stream_obj* s,
   struct op_storage* storage = &s->storage;
   /* add new op at the beginning of the linked list. The memory is freed
   in remove_from_storage */
-  struct op_and_state* new_op = static_cast<struct op_and_state*>(
-      gpr_malloc(sizeof(struct op_and_state)));
-  memcpy(&new_op->op, op, sizeof(grpc_transport_stream_op_batch));
-  memset(&new_op->state, 0, sizeof(new_op->state));
-  new_op->s = s;
-  new_op->done = false;
+  op_and_state* new_op = grpc_core::New<op_and_state>(s);
```

Comment:
This was not plain initialization to 0; some of the data in `op` was copied to `new_op`.

Reply:
Thank you for running the tests! those I have already moved to the ctor on line 325-326. Both `s` and `next` are properly set.

Reply:
Sorry I should have referred to the deleted lines. I am referring to line 319 of the old code, `memcpy`. Basically the information in `op` needs to be copied to `new_op`.

Reply:
Oh I see. Indeed. Thank you very much for catching this!!